workspace_tools/targets.py

```diff
@@ -254,6 +254,13 @@ def __init__(self):
         self.extra_labels = ['NXP', 'LPC23XX']
         self.supported_toolchains = ["ARM", "GCC_ARM", "GCC_CR"]
 
+class LPC2460(LPCTarget):
```

Comment:
It is MCU, of course. I did everything by analogy with LPC2368(similar MCU). Do I need to do some renaming?